pcm_win32.cpp

```diff
@@ -127,7 +125,7 @@ void PcmOutputWin32::wait()
         return;
     }
     while ((buffers[buftail].hdr.dwFlags & WHDR_DONE) == 0) {
-        Sleep(100);
+        Sleep(0);
```

Comment:
I will not accept any code containing a Sleep(0) without robust justification. What's the problem? How does this fix it? What are the side effects?